demo/llama_stack_mcp_agent.py

```
@@ -0,0 +1,115 @@
+from llama_stack_client.lib.agents.agent import Agent
+from llama_stack_client.lib.agents.event_logger import EventLogger
+from llama_stack_client.types.tool_group import McpEndpoint
+from llama_stack_client import LlamaStackClient
+from termcolor import cprint
```

Ruff (0.12.2)

`termcolor.cprint` imported but unused

Remove unused import: `termcolor.cprint`

(F401)